examples/hello-world/hello-numpy-cross-val/jobs/hello-numpy-cross-val-only-list-models/app/config/config_fed_server.json

```diff
@@ -0,0 +1,44 @@
+{
+  "format_version": 2,
+  "model_dir": "{$SERVER_MODEL_DIR}",
+  "server": {
+    "heart_beat_timeout": 600
+  },
+  "task_data_filters": [],
+  "task_result_filters": [],
+  "components": [
+    {
+      "id": "persistor",
```

Comment:
Do you still need the persistor in this example? I don't see it used anywhere.

Reply:
No need "persistor". Removed.